string name
string color
float32 r
float32 g
float32 b
float32 duration
bool wait
---
